modules/reactor/test/tests/meshgenerators/reporting_id/cartesian_id/assembly_reporting_id_cell.i

```diff
@@ -0,0 +1,62 @@
+[Mesh]
+  [pin1]
+    type = ConcentricCircleMeshGenerator
+    num_sectors = 4
```

Comment:
can you drop to less sectors or less rings? gold file size are proportional to that